test/pipelines/pipelines-it-installation.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
 - job: 'pip_install_macOS_python36'
   pool:
-    vmImage: 'macOS-10.13'
+    vmImage: 'macOS-latest'
   strategy:
     matrix:
```

Comment:
the `strategy` section can be removed,  it does not take effect as we expected.
The actual default python version depends on the image.

Reply:
updated, thanks!